benchmarking/README.md

```diff
@@ -66,9 +66,9 @@ https://papers.nips.cc/paper/2003/file/49d4b2faeb4b7b9e745775793141e2b2-Paper.pd
 For example,
 
 ```sh
-$ ./collect_LIBSVM.sh /tmp ~/LIBSVM
-$ julia --project=. generate_l1_svm_lp.jl --input_filename=~/LIBSVM/duke \
-	  --output_filename=~/LIBSVM/duke.mps.gz --regularizer_weight=1.0
+$ ./collect_LIBSVM.sh $HOME/LIBSVM
```

Comment:
We should try to use "good" style even in example scripts when possible, so "${HOME}/..."